Makefile

```diff
@@ -110,6 +110,7 @@ $(DOCKER_TARGETS): docker-%:
 		--rm \
 		-e GOCACHE=/tmp/.cache/go \
 		-e GOMODCACHE=/tmp/.cache/gomod \
+		-e GOLANGCI_LINT_CACHE=/tmp/.cache/golangci-lint \
```

Comment:
I am curious, are you running into any issues by running this? if so which one?

Reply:
We are using native golangci-lint action, so I guess this is only for local devel, right? https://github.com/NVIDIA/nvidia-container-toolkit/blob/main/.github/workflows/golang.yaml#L33

Reply:
```➜  container-toolkit git:(main) ✗ make .shell
docker run \
        --rm \
        -ti \
        -e GOCACHE=/tmp/.cache/go \
        -e GOMODCACHE=/tmp/.cache/gomod \
        -v /Users/elezar/src/container-toolkit:/work \
        -w /work \
        --user $(id -u):$(id -g) \
        ghcr.io/nvidia/k8s-test-infra:devel-go1.20.5
I have no name!@bf871339fa93:/work$ make check
golangci-lint run ./...
2024/03/15 06:48:53 failed to initialize build cache at /.cache/golangci-lint: mkdir /.cache: permission denied
make: *** [Makefile:91: golangci-lint] Error 1
I have no name!@bf871339fa93:/work$```